build/debian/changelog

```diff
@@ -1,3 +1,9 @@
+mixxx (2.1.3-0ubuntu1) bionic; urgency=medium
```

Comment:
@Be-ing : shouldn't it be 2.1.4-0ubuntu1 here?

Reply:
good catch, thanks. Fixed in a force pushed commit.